maintenance/importConfig.ts

```diff
@@ -441,7 +441,7 @@ async function parseOzwProduct(
 		if (param.type === "bitset") {
 			const bitSetIds = ensureArray(param.BitSet);
 			for (const bitSet of bitSetIds) {
-				let id = parseInt(bitSet.id).toString(16);
+				let id = (parseInt(bitSet.id) - 1).toString(16);
```

Comment:
No, we need bit shifts! The thing in the square brackets is the bit *mask* itself, e.g. `0xc` to target bits 2 and 3 (zero-based)

Reply:
Arrggg I feel so stupid. Sorry

Reply:
@AlCalzone Check it now :)